layouts/default.vue

```diff
@@ -1,19 +1,64 @@
 <template>
   <div class="app">
-    <div class="app__header">
+    <div :class="['app_header', {'app__header__open' : this.header }]">
       <h1 class="app__header__title">Unicorns</h1>
-      <back-button v-if="!isHomepage" />
+      <back-button v-if="!isHomepage" @close-menu="closeMenu"/>
+      <menu-bouton @action-menu="actMenu"/>
+      <div class="app__header__menu-title" v-if="isOpen">
+       <item-menu v-for="(items,index) in itemsMenu" :item="items" :key="index" @nav-menu="navMenu"/>
+      </div>
     </div>
     <Nuxt />
+
+    <fab-add v-if="isHomepage" @close-menu="closeMenu"/>
```

Comment:
finalement tu souhaites que le fab-add soit sur toutes les pages utilisant le default layout ?

Reply:
il est affiché que sur l'homepage parce par exemple si on ouvre le menu sur l'homepage et qu'on clique sur la bulle, quand on arrive sur la page create alors le menu ne se ferme pas, j'ai donc fais un emit dans fab quand on clique dessus alors il ferme le menu sinon je ne trouvais pas d'autres solutions pour detecter le clique et modifier la valeur this.menuOpen